editoast/editoast_core/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+name = "editoast_core"
+version.workspace = true
+edition.workspace = true
+license.workspace = true
+
+[features]
+mocking_client = []
+
+[dependencies]
+approx.workspace = true
+chrono.workspace = true
+deadpool.workspace = true
+derivative.workspace = true
+editoast_common = { workspace = true }
+editoast_schemas = { workspace = true }
+futures-util.workspace = true
+hostname.workspace = true
+http = "1.3.1"
+itertools.workspace = true
+lapin.workspace = true
+opentelemetry.workspace = true
+reqwest.workspace = true
+serde = { workspace = true, features = ["derive"] }
+serde_json.workspace = true
+thiserror.workspace = true
+tokio.workspace = true
+tracing.workspace = true
+tracing-opentelemetry.workspace = true
+url.workspace = true
+utoipa.workspace = true
+uuid.workspace = true
+
+[dev-dependencies]
```

Comment:
You need to add `osrd_schemas` with the feature `testing` in this section. Take a look at https://github.com/OpenRailAssociation/osrd/pull/11852.

Reply:
I merged the other PR #11852, so I'm resolving this thread. Please squash the commits from my PR in the right commits of your PR.